spec/configuration_spec.rb

```diff
@@ -33,5 +33,19 @@
         expect(config.tracked_files).to be_nil
       end
     end
+
+    describe "#minimum_coverage" do
+      it "warns you about your usage" do
+        expect(config).to receive(:warn).with("The coverage you set for minimum_coverage is greater than 100%")
+        config.minimum_coverage(100.01)
```

Comment:
Thanks for the tests! We're missing a negative test though. Aka 100.0 doesn't trigger a warning (current tests would pass if we just warned all the time it seems like to me)